release-notes/6.0/preview/api-diff/preview5/.Net/6.0-preview5_System.Runtime.CompilerServices.md

```diff
@@ -0,0 +1,10 @@
+# System.Runtime.CompilerServices
+
+``` diff
+ namespace System.Runtime.CompilerServices {
+     public static class RuntimeFeature {
++        public const string VirtualStaticsInInterfaces = "VirtualStaticsInInterfaces";
```

Comment:
CC. @jeffhandley that this made Preview 5 for the generic-math work.

Reply:
Ah awesome, I didn't realize this went it already!